master/internal/rm/kubernetesrm/pods.go

```diff
@@ -77,8 +79,8 @@ type pods struct {
 	loggingTLSConfig model.TLSClientConfig
 	loggingConfig    model.LoggingConfig
 
-	informers                    []*actor.Ref
-	nodeInformer                 *actor.Ref
+	informers                    []*informer
+	nodeInformer                 *nodeInformer
```

Comment:
are these used?

Reply:
good catch, informers is, but nodeInformer isn't.